third_party/3/jwt/contrib/algorithms/__init__.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Callable, Any, Union
+from hashlib import _DataType
+
+# In reality, _HashAlg is a function of the type
+# that hashlib.sha256/384/512 are, but there doesn't
+# seem to be a consistent exportable name that we can reference
+# for that, so we'll just say it's a Callable here.
```

Comment:
Hey, I got an email notification where you said you were able to reproduce my bug and were debugging it, but visiting this page now, I'm not seeing it--did you manage to make any progress?

I'll try to test my stub in the context of the `typeshed` repo itself and see if that helps--so far I have been writing it in [my own independent repository](https://github.com/toolness/mypy-experiments/tree/master/pyjwt-stubs) because I need unit tests to make sure the annotations align with the example snippets in pyJWT's own documentation, but I understand from #862 that those tests wouldn't be useful as a contribution to this repository, so I've kept them separate. But this also means I've mostly been testing the stubs from my repository, which doesn't have the intermediary `3` directory that might make mypy happy.

Will try to get this working on my end soon! Sorry for the delay.